pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -364,6 +364,30 @@ func setSupportedSubsystems(cgroupConfig *libcontainerconfigs.Cgroup) error {
 	return nil
 }
 
+// getCpuWeight converts from the range [2-262144] to [1-10000]
```

Comment:
nit: I guess it is just a preference, but I am used to writing closed intervals as `[2, 262144]` as I find it easier to read. Not 100% sure what is used elsewhere tho..

Reply:
sure, that is more readable.  Pushed an updated version.  Thanks for the quick review!